binding/HarfBuzzSharp.Shared/FontFunctions.cs

```diff
@@ -0,0 +1,536 @@
+		private static readonly GlyphOriginProxyDelegate GlyphOriginProxy = GlyphOriginProxyImplementation;
+		private static readonly GlyphKerningProxyDelegate GlyphKerningProxy = GlyphKerningProxyImplementation;
+		private static readonly GlyphExtentsProxyDelegate GlyphExtentsProxy = GlyphExtentsProxyImplementation;
+		private static readonly GlyphContourPointProxyDelegate GlyphContourPointProxy =
+			GlyphContourPointProxyImplementation;
+		private static readonly unsafe GlyphNameProxyDelegate GlyphNameProxy = GlyphNameProxyImplementation;
+		private static readonly GlyphFromNameProxyDelegate GlyphFromNameProxy = GlyphFromNameProxyImplementation;
+
+		public delegate bool FontExtentsDelegate (Font font, object fontData, out FontExtents extents);
+		public delegate bool NominalGlyphDelegate (Font font, object fontData, uint unicode, out uint glyph);
+		public delegate bool VariationGlyphDelegate (Font font, object fontData, uint unicode, uint variationSelector,
+			out uint glyph);
+		public delegate uint NominalGlyphsDelegate (Font font, object fontData, uint count,
+			ReadOnlySpan<uint> codepoints, Span<uint> glyphs);
+		public delegate int GlyphAdvanceDelegate (Font font, object fontData, uint glyph);
+		public delegate void GlyphAdvancesDelegate (Font font, object fontData, uint count, ReadOnlySpan<uint> glyphs,
+			Span<int> advances);
+		public delegate bool GlyphOriginDelegate (Font font, object fontData, uint glyph, out int x, out int y);
+		public delegate int GlyphKerningDelegate (Font font, object fontData, uint firstGlyph, uint secondGlyph);
+		public delegate bool GlyphExtentsDelegate (Font font, object fontData, uint glyph, out GlyphExtents extents);
+		public delegate bool GlyphContourPointDelegate (Font font, object fontData, uint glyph, uint pointIndex,
+			out int x, out int y);
+		public unsafe delegate bool GlyphNameDelegate (Font font, object fontData, uint glyph, char* nameBuffer, int size);
+		public delegate bool GlyphFromNameDelegate (Font font, object fontData, string name, out uint glyph);
+
+		public FontFunctions () : this (IntPtr.Zero)
+		{
+			Handle = HarfBuzzApi.hb_font_funcs_create ();
+		}
+
+		internal FontFunctions (IntPtr handle) : base (handle) { }
+
+		internal FontFunctions (IntPtr handle, bool zero) : base (handle, zero) { }
+
+		public static FontFunctions Empty => emptyFontFunctions.Value;
+
+		internal Font Font { get; set; }
+
+		internal object FontData { get; set; }
+
+		public bool IsImmutable => HarfBuzzApi.hb_font_funcs_is_immutable (Handle);
+
+		public void MakeImmutable () => HarfBuzzApi.hb_font_funcs_make_immutable (Handle);
+
+		[MonoPInvokeCallback (typeof (FontExtentsProxyDelegate))]
+		private static bool FontExtentsProxyImplementation (IntPtr font, IntPtr fontData, out FontExtents extents, IntPtr context)
+		{
+			var multi = DelegateProxies.Get<GetMultiDelegateDelegate> (context, out _);
+			var del = (FontExtentsDelegate)multi.Invoke (typeof (FontExtentsDelegate));
```

Comment:
Just pushed a commit to master that will compress these lines:

```csharp
 var del = GetMulti<FontExtentsDelegate> (context, out _);  
```

Reply:
Will make the needed changes 👍